controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
@aslakknutsen I was wrong. I have modified my code ->

https://github.com/almighty/almighty-core/pull/804/files#diff-55f2da878ed2fcb42e5373010f02ff92R157

Does this make sense?

If not, please tell me how i can fix it.